<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnWidget</class>
 <widget class="QWidget" name="ColumnWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="containerLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="moveButtonsLayout">
       <item>
        <widget class="QPushButton" name="moveLeftButton">
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveRightButton">
         <property name="text">
          <string>→</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="newNoteButton">
       <property name="text">
        <string>New Note</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="notesLayout">
       <property name="spacing">
        <number>10</number>
       </property>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
